exercises/perfectShuffle.js

```diff
@@ -9,6 +9,17 @@
 
 function perfectShuffle(array) {
   // This is your job. :)
+  let i = array.length;
```

Comment:
I'd give this a more evocative name, like

```javascript
let mid = array.length / 2
```

When I see `i`, I think it's a counter/index variable (and there's a good chance you'll do the same unintentionally).